public/styles.css

```diff
@@ -356,13 +353,18 @@ a {
 /* Main content styling */
 .main-content {
     margin-left: 210px; /* Same as the width of the sidenav */
-    padding: 20px;
+    margin-right: 0px;
+    padding-bottom: 100px;
+    padding-top: 100px;
+    /* width: 37vh; */ /* I'm not setting width here because the parent div sends in a weird black bar on the right in all child divs*/
+    height: 90vh;
 }
 
 /* Hidden by default */
 .tabcontent {
     display: none;
```

Comment:
This moves the `tabcontent`, which is a child under the `main-content`, and allows for a little space at the top and the bottom of the parent div. If this is removed, the lists in `tabcontent` will start from the TOP of the view. 

if you don't like the positions of the tab contents, I would highly recommend to have each of your `sidenav` items in its own div, and use THAT to center it, instead of using the class `tabcontent` to move them around, since you would be applying settings to a smaller subset of items that way.